sbnobj/Common/Reco/SimpleFlashMatchVars.h

```diff
@@ -0,0 +1,45 @@
+#ifndef sbncode_Simpleflashmatchvars_H
+#define sbncode_Simpleflashmatchvars_H
+
+#include "TVector3.h"
+
+namespace sbn
+{
+  class SimpleFlashMatch
+  {
+  public:
+  SimpleFlashMatch(
+    bool present = false, double time = -1,
+    double charge_q = -1, double light_pe = -1,
+    double score = -1, double scr_y = -1, double scr_z = -1,
+    double scr_rr = -1, double scr_ratio = -1,
+    TVector3 chargeXYZ = TVector3(-999, -999, -999),
+    TVector3 lightXYZ = TVector3(-999,-999,-999)):
+    mPresent(present),
+    mTime(time),
+    mChargeQ(charge_q),
+    mLightPE(light_pe),
+    mScore(score),
+    mScr_y(scr_y),
+    mScr_z(scr_z),
+    mScr_rr(scr_rr),
+    mScr_ratio(scr_ratio),
+    mChargeXYZ(chargeXYZ),
+    mLightXYZ(lightXYZ)
+    {}
+
+    bool mPresent;
+    double mTime;
+    double mChargeQ;
+    double mLightPE;
+    double mScore;
+    double mScr_y;
+    double mScr_z;
+    double mScr_rr;
+    double mScr_ratio;
+    TVector3 mChargeXYZ;
```

Comment:
I know lots of people have Opinions about TVector3, not without justification. There are larsoft 3-vector types that are supposed to be better. Can you use one of those?

Reply:
Good point.
How about using `geo::Point_t` from `#include "larcoreobj/SimpleTypesAndConstants/geo_vectors.h" `?

Reply:
Come to think about it. Whilst I'm all for removing `Tvector3`, there are several other instances of it on `sbnobj`. 
Better to make one sweep change in another PR.